src/analyses/callstringTracking.ml

```diff
@@ -0,0 +1,36 @@
+open Analyses
```

Comment:
Might be good to put a reference to S. et al "Clustered Relational Thread-Modular Abstract Interpretation with Local Traces" where this idea came from (at least to us), or some other paper that proposed this.